include/aikido/common/EnumFlags.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef AIKIDO_COMMON_ENUMFLAGS_HPP_
+#define AIKIDO_COMMON_ENUMFLAGS_HPP_
+
+#include <type_traits>
+
+/// Enable bitwise operators for strongly-typed enums.
+///
+/// Adapted from
+/// http://blog.bitwigglers.org/using-enum-classes-as-type-safe-bitmasks/
```

Comment:
Nit: Maybe we want to make this as non-doxygen style comment because this comment is not specifically tied to a code but just a note?